include/iotc_jwt.h

```diff
@@ -23,58 +23,48 @@
 extern "C" {
 #endif
 
+/*! \file
+ * @brief Creates JSON Web Tokens.
```

Comment:
+ The Clout IoT uses JWT for client authentication. The SDK generates the required JWT with the functions in this header.

Reply:
Done